lib/pipedrive/deals.ex

```diff
@@ -26,19 +26,22 @@ defmodule Pipedrive.Deals do
   """
   @impl Pipedrive.RESTEntity
   @spec create(map(), Keyword.t()) :: API.response()
-  def create(%{title: _} = body_params, opts \\ []) do
+  def create(body_params, opts \\ [])
+  def create(%{title: _} = body_params, opts), do: do_create(body_params, opts)
```

Comment:
it sounded like you didn't want to accept atom keys anymore, but here they are. did i misunderstand your intention?

Reply:
Long story short, Mihai forced me to bring it back 😁. My intention was not to care what the keys are, I don't have an opinion about whether this kind of check of required keys should be here or not, and Mihai wanted the check back. Thus the pattern-matching on both string and atom keys.